api_client/python/timesketch_api_client/scenario.py

```diff
@@ -103,19 +165,119 @@ def __init__(self, sketch_id, question_id, uuid, api):
         self.uuid = uuid
         self.api = api
         self.sketch_id = sketch_id
+        self._name = None
+        self._display_name = None
+        self._description = None
+        self._data = None
         super().__init__(
             api=api, resource_uri=f"sketches/{self.sketch_id}/questions/{self.id}/"
         )
 
+    def _update(self, data):
+        """Internal helper to update the question."""
+        resource_url = f"{self.api.api_root}/{self.resource_uri}"
+        response = self.api.session.post(resource_url, json=data)
+        self._data = error.get_response_json(response, logger)
+        updated_object = self._data["objects"][0]
+        self._name = updated_object["name"]
```

Comment:
```suggestion
        self._name = updated_object.get("name", self._name)
        self._display_name = updated_object.get("display_name", self._display_name)
        self._description = updated_object.get("description", self._description)
```

Have a safety net in case the values are not present in the `update_object`